governance/polls/Ratification Poll for Offboarding Events Core Unit (EVENTS-001) - August 8, 2022.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Ratification Poll for Offboarding Events Core Unit (EVENTS-001) - August 8, 2022
+summary: This proposal is to offboard the Events Core Unit (EVENTS-001)
+discussion_link: https://forum.makerdao.com/t/mip40c3-sp73-modify-core-unit-budget-events-001/15991
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-08-08T16:00:00
+end_date: 2022-08-22T16:00:00
+---
+# Ratification Poll for Offboarding Events Core Unit (EVENTS-001) - August 8, 2022
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, August 8 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to Offboarding Events Core Unit (EVENTS-001) being accepted and implemented in the Maker Protocol.
+
+### MIP40c3-SP73
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*This Core Unit is authorized to spend $898,150 annually. Events are a “nice to have” and not required for this bear market, particularly considering how Daivinity was executed. Mkr token holders’ money was spent on unknown attendees and no business deals or new hires have been credited to that event.*
+
+*Given current budget constraints, the protocol simply does not accrue value from a core unit whose mission statement is: “To provide the best MakerDAO branded event experience to the crypto & Maker community including industry professionals, influencers, and enthusiasts.”*
```

Comment:
I suggest placing quotation marks around these, in addition to italicizing. (Hey, @prose11 , maybe we can use blockquotes now?)